lisa/sut_orchestrator/azure/common.py

```diff
@@ -488,13 +495,38 @@ def get_storage_credential(
     return {"account_name": account_name, "account_key": key.value}
 
 
-def get_or_create_storage_container(
+def generate_sas(
```

Comment:
`generate_sas_token` as the returned value name.

Reply:
updated